book/javascript/highlightjs-gleam.js

```diff
@@ -74,10 +73,22 @@ hljs.registerLanguage("gleam", function (hljs) {
           },
         ],
       },
+      {
+        className: "attribute",
```

Comment:
This needs to go into the general highlight js and into the compiler for the docs gen, right? Could we submodule this?

Reply:
https://github.com/gleam-lang/gleam-highlight.js/pull/4

Reply:
https://github.com/gleam-lang/gleam/pull/2267